lib/xdrgen/generators/java/XdrString.erb

```diff
@@ -0,0 +1,57 @@
+package <%= @namespace %>;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.Arrays;
+
+public class XdrString implements XdrElement {
+    private byte[] bytes;
+    private String text;
```

Comment:
I'm not a fan of keeping the same state in two variables. I'd get rid of the `text` member and just keep the `bytes`, as that's the closest representation to what's on the wire.